### DISCUSSION TITLE
Would love to support a plugin

### DISCUSSION THREAD

**Discussion by soilhare** [streamlink/streamlink#5879] [Q&A]
### Description

I have a favorite streamer who was broadcast live on Twitch before, but not long ago she switched to another platform, www.flextv.co.kr, but I found that streamlink does not support this platform, I also searched the previous discussion, and found that there are other people who want this, I hope the developer can help support it, although this platform has 19+ content, but you need to log in to the account and go through adult authentication before you can watch 19+, as far as I know, this certification is difficultIt's also troublesome, I just want to see the streamer I like, maybe it won't subvert the developer's original intention of not wanting to support adult content

**Comment by bastimeyer** [comment]
1. https://github.com/streamlink/streamlink/blob/master/.github/DISCUSSION_TEMPLATE/q-a.yml
2. https://github.com/streamlink/streamlink/issues?q=flextv+label%3A%22plugin+request%22+sort%3Aupdated-desc
3. https://github.com/streamlink/streamlink/discussions?discussions_q=flextv
